app/views/pages/moving-guide/reimbursements.html.erb

```diff
@@ -1,11 +1,122 @@
+    </div>
+
+    <div id="temporary-lodging-allowance-oconus-moves">
+      <h2>Temporary Lodging Allowance (TLA) <%= abbr_tag('oconus') %> Moves</h2>
+
+      <p>Use this allowance towards temporary lodging and meals while you wait for housing at your new duty station.</p>
+      <p>Since <%= abbr_tag('oconus') %> household shipments take longer to arrive, you can use <%= abbr_tag('tla') %> for a maximum of 60 days upon arrival at your new duty station and 10 days prior to departure.</p>
+      <p>The amount you’ll get reimbursed is calculated based on a percentage of the per diem for lodging in your host country.</p>
+      <p>In addition to lodging and meals, you can also get reimbursed for currency conversion fees.</p>
+      <p>If necessary, you can get a <%= abbr_tag('tla') %> advance payment.</p>
+    </div>
+
+    <div id="per-diem-for-pcs-travel">
+      <h2>Per Diem for <%= abbr_tag('pcs') %> Travel</h2>
+
+      <p>This serves as your reimbursement for meals and lodging while you are on your way to your new duty station.</p>
+      <p><b>Travel by Privately Owned Conveyance:</b> The flat Standard <%= abbr_tag('conus') %> rate, which combines both Lodging and Meals/Incidentals. For example, the rate on 10/1/2017 is $144.</p>
+      <p><b>Travel by Commercial Means:</b> The ‘Lodgings-Plus’ rate for the new PDS or the rate for the delay point if the member stops overnight.</p>
+      <p><b>Dependent:</b> 75% of the member's applicable rate for each dependent age 12 or older, and 50% for each dependent under age 12.</p>
+    </div>
+
+    <div id="mileage-rates">
+      <h2>Mileage Rates</h2>
+
+      <p>Get paid for the miles you put on your personal vehicle when you drive to your new duty station. This includes to motorcycles, and personally owned aircraft as well. Additionally, if you have a <%= abbr_tag('tdy') %> en route to your next duty station and you drive your personal vehicle to that <%= abbr_tag('tdy') %>, you will be paid at the <%= abbr_tag('tdy') %> rate.</p>
+      <p>For example: As of 01 Jan 2017, the per mile rate is $0.17. <%= link_to raw("Find Your Mileage Rate for your #{abbr_tag('pcs')}."), 'https://www.defensetravel.dod.mil/site/otherratesMile.cfm', class: 'usa-external_link' %></p>
+    </div>
+
+    <div id="relocation-income-tax-allowance">
+      <h2>Relocation Income Tax (RIT) Allowance</h2>
+
+      <p>While for most service members this will not apply, if there were expenses incurred as a result of a Permanent Change of Station that you were not reimbursed for, may be able to deduct them from on your income taxes.</p>
+      <p>For more information, visit the Internal Revenue Service (IRS) Website and search for Publication 3, Armed Forces’ Tax Guide.</p>
+    </div>
+
+    <div id="pets">
+      <h2>Pets</h2>
+
+      <p>You can get reimbursed for any mandatory quarantine fees up to $550 per move, for dogs and cats only.</p>
+      <p><b>Heads Up:</b> There is no reimbursement for transportation fees.</p>
```

Comment:
This is saying that there is no reimbursement for _pet_ transportation fees, right?